src/Module/BaseApi.php

```diff
@@ -158,6 +177,55 @@ protected static function getCurrentUserID()
 		return (int)self::$current_user_id;
 	}
 
+	private static function checkBearer(string $authorization)
+	{
+		return(strpos($authorization, 'Bearer ') !== false);
```

Comment:
Shouldn't the authorization string start with "Bearer"?

Reply:
Right.

Reply:
Changed